systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -918,4 +918,12 @@ public static void checkCOlogForUsedVariable(String varName) {
         assertTrue(coLog.contains("User defined container template environment variable " + varName + " is already in use and will be ignored"));
         LOGGER.info("ClusterOperator logs contains proper warning");
     }
+
+    public static void waitUntilPodContainersCount(String podNamePrefix, int numberOfContainers) {
+        LOGGER.info("Waiting till pod {} will have {} containers", podNamePrefix, numberOfContainers);
+        TestUtils.waitFor("Waiting till pod" + podNamePrefix + " will have " + numberOfContainers + " containers",
+                Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_STATUS_TIMEOUT,
+            () -> kubeClient().listPodsByPrefixInName(podNamePrefix).get(0).getSpec().getContainers().size() == numberOfContainers);
```

Comment:
Is 3 minutes enough? Since from time to time the list of pods contained TO so I expect, one additional reconciliation was needed.

Reply:
I experimented with it(execute this test 6 times in a row) and came to a conclusion that is enough.